### ISSUE TITLE
remote command does not work from sensor trigger

### ISSUE THREAD

**Issue by BNNorman** [connected-little-boxes/box-code#6]
Tried this on CLB-d122dc which only has a button
```
{"process":"servo","command":"setservopos","value":0.5,"sensor":"button","trigger":"pressed","to":"CLB-7ab3cf"}
```
The pressed the button and - nothing.

The direct command does work:-
```
{"process":"servo","command":"setservopos","value":0.5,"to":"CLB-7ab3cf"}
```

I monitored the MQTT topic lb/command/CLB-7ab3cf but there was nothing to be seen,

**Comment by CrazyRobMiles** [comment]
I can't replicate this. What other devices are connected to the box?

**Comment by BNNorman** [comment]
None. It just has a button. I will have to build another device with a full set of sensors.

Didn't the meetup show two slashes in the mqtt topic?

**Comment by BNNorman** [comment]
Just setup a device (ESP32 CLB-a0e350cc) with just a servo and it works with my test device which has a number of sensors using these commands.

{"process":"servo","command":"setservopos","value":1,"sensor":"button","trigger":"pressed","to":"CLB-a0e350cc"}
{"process":"servo","command":"setservopos","value":0,"sensor":"button","trigger":"released","to":"CLB-a0e350cc"}

I tried again with the button-only ESP8266 device and it didn't work. Used node red to monitor the lb/# topic and saw nothing at all. I reflashed the ESP8266 and made sure it was version 1.0.0.95 - still didn't work. I'll investigate more when I've built a second device with all bells and whistles.
